codegen-server/build.gradle.kts

```diff
@@ -4,10 +4,12 @@
  */
 
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.jreleaser.model.Active
 
 plugins {
     kotlin("jvm")
     `maven-publish`
+    id("org.jreleaser") version "1.11.0"
```

Comment:
i think we will also need to add and configure the `signing` plugin.

Reply:
What is jreleaser giving us here?

Reply:
We don't need it - JReleaser will sign the artifacts in staging